PENDING.md

```diff
@@ -0,0 +1,6 @@
+# Unreleased Changes
```

Comment:
Nit: I think I would have called that file `UNRELEASED.md`

Reply:
I've changed it to CHANGES_UNRELASED.md. I agree UNRELEASED is better, but I want it to be next to the CHANGELOG file in the file list.